bundles/AdminBundle/Resources/public/js/pimcore/asset/helpers/grid.js

```diff
@@ -199,7 +199,9 @@ pimcore.asset.helpers.grid = Class.create({
                         locked: this.getColumnLock(field),
                         renderer: function (value) {
                             if (value) {
-                                return '<img style="max-width: 200px; max-height: 100px;" src="' + value + '" loading="lazy">';
+                                return  '<div class="list-img-container">' +
+                                    '<img style="max-width: 150px; max-height: 100px; object-fit: contain;" src="' + value + '" loading="lazy">'+
```

Comment:
You can add this styles also to the admin.css `.list-img-container > img`

Reply:
you are right.
Done